.github/workflows/kaas-docker-image.yml

```diff
@@ -0,0 +1,34 @@
+name: Docker image
+on:
+  push:
```

Comment:
How about we follow the same rules as in https://github.com/opencybersecurityalliance/kestrel-lang/blob/develop/.github/workflows/publish-to-pypi.yml#L4 to trigger this---release a new container image when we do a new kestrel-lang release?

Reply:
ok done